pom.xml

```diff
@@ -268,10 +268,11 @@
         <version.io.opentracing.tracerresolver>0.1.5</version.io.opentracing.tracerresolver>
         <version.io.opentracing.servlet>0.1.0</version.io.opentracing.servlet>
         <version.io.reactivex.rxjava>2.2.2</version.io.reactivex.rxjava>
+        <version.io.rest-assured>3.0.0</version.io.rest-assured>
         <version.io.prometheus.simpleclient>0.6.0</version.io.prometheus.simpleclient>
         <version.io.smallrye.smallrye-config>1.3.6</version.io.smallrye.smallrye-config>
         <version.io.smallrye.smallrye-health>1.0.2</version.io.smallrye.smallrye-health>
-        <version.io.smallrye.smallrye-metrics>1.1.3</version.io.smallrye.smallrye-metrics>
+        <version.io.smallrye.smallrye-metrics>1.1.0</version.io.smallrye.smallrye-metrics>
```

Comment:
is this downgrade change intentional?

Reply:
@jstourac it is not a downgrade. The smallrye-metrics component changes its GAV convention, we upgraded from `io.smallrye:smallrye-metrics:1.1.3` (that covers MP Metrics 1.x) to `io.smallrye:smallrye-metrics-2.0:1.1.0` (that covers MP Metrics 2.x).

I updated the `artifactId` in the pom.xml but did not change the name of the property that sets the version. sorry for the confusion :)